@codex please fix

```
> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
 ✓ tests/eslintPluginPresence.test.ts (1 test) 7ms
 ✓ frontend/tests/gameStateFlush.test.ts (1 test) 5ms
 ✓ tests/checkoutFetchDepth.test.ts (1 test) 6ms
 ✓ tests/npmTestRunsAllSuites.test.ts (1 test) 5ms
 ✓ tests/astroConfigLogLevel.test.ts (1 test) 4ms
 ✓ tests/prismSvelte.test.ts (1 test) 17ms
 ✓ scripts/tests/dbBenchmarkScript.test.ts (1 test) 7ms
 ✓ tests/progressionBalance.test.ts (1 test) 43ms
 ✓ tests/domException.test.ts (1 test) 7ms
 ✓ tests/indexeddb.server.test.ts (1 test) 290ms
 ✓ frontend/tests/sanity.test.ts (1 test) 3ms
[warn] src/pages/process/[slug]/__tests__/ProcessView.spec.ts
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
 ❯ scripts/tests/prettierFormatting.test.ts (1 test | 1 failed) 18994ms
   × Prettier formatting > frontend sources are formatted 18992ms
     → Command failed: npm run format:check

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  scripts/tests/prettierFormatting.test.ts > Prettier formatting > frontend sources are formatted
Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5
      6|   test('frontend sources are formatted', () => {
      7|     const frontendDir = path.join(__dirname, '../../frontend');
      8|     execSync('npm run format:check', {
       |     ^
      9|       cwd: frontendDir,
     10|       stdio: 'inherit',

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯


 Test Files  1 failed | 290 passed (291)
      Tests  1 failed | 1026 passed (1027)
   Start at  06:51:16
   Duration  167.68s (transform 10.40s, setup 11.16s, collect 45.76s, tests 110.75s, environment 189.59s, prepare 36.00s)


Error: Error: Command failed: npm run format:check
 ❯ scripts/tests/prettierFormatting.test.ts:8:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { status: 1, signal: null, output: [ null, null, null ], pid: 9329, stdout: null, stderr: null }

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/src/pages/process/[slug]/__tests__/ProcessView.spec.ts
+++ b/frontend/src/pages/process/[slug]/__tests__/ProcessView.spec.ts
@@ -262,7 +262,7 @@ describe('ProcessView detail controls', () => {
             'energy-item': 9000,
         };
         getItemCountMock.mockImplementation((itemId: string) => counts[itemId] ?? 0);
-        buyItemsMock.mockImplementation((plannedBuys: Array<{id: string; quantity: number}>) => {
+        buyItemsMock.mockImplementation((plannedBuys: Array<{ id: string; quantity: number }>) => {
             plannedBuys.forEach(({ id, quantity }) => {
                 counts[id] = (counts[id] ?? 0) + quantity;
             });

